x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/parser/ExpressionTests.java

```diff
@@ -221,4 +221,25 @@ public void testInEmptySet() {
         expectThrows(ParsingException.class, "Expected syntax error",
             () -> expr("name in ()"));
     }
+
+    public void testChainedComparisons() {
```

Comment:
Can you add a test for something like `1 * 2 <= 3 * 4`, just to make sure these operators are still playing nicely with precedence for other operators. Even better would be to verify the AST matches this: `(1 * 2) <= (3 * 4)`